Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMWidevineDecryptorGStreamer.cpp

```diff
@@ -0,0 +1,222 @@
+/* GStreamer OpenCDM decryptor
+ *
+ * Copyright (C) 2016-2017 TATA ELXSI
+ * Copyright (C) 2016-2017 Metrological
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public
+ * License along with this library; if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Suite 500,
+ * Boston, MA 02110-1335, USA.
+ */
+
+#include "config.h"
+#include "WebKitOpenCDMWidevineDecryptorGStreamer.h"
+
+#if ENABLE(LEGACY_ENCRYPTED_MEDIA) && USE(GSTREAMER) && USE(OCDM)
+
+#include "GUniquePtrGStreamer.h"
+#include <gst/gst.h>
+#include <open_cdm.h>
+#include <stdio.h>
+#include <string>
+#include <wtf/glib/GUniquePtr.h>
+#include <wtf/text/WTFString.h>
```

Comment:
I think these include need also some "compiler purge", specially <string> which I couldn't find where it is used.